app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -169,13 +170,19 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
         return await _getAddressInfos(addresses.slice(beginIndex, beginIndex + gapLimit))
       })
     )
-    const addressTokenBundles = txHistory.map((txHistoryEntry) => {
+    const validTxHistory = txHistory.map((txHistoryEntry) => {
```

Comment:
we use just the balance from the address info response, so if I'm not mistaken there's no reason to filter the tx history as we don't do anything with it here

I think that the names are just misleading and what we call "txHistory" here should rather be "addressInfos"

Here's a sample response I looked up:

```
{"Right":{"caAddresses":[<addresses requested>],"caTxNum":248,"caBalance":{"getCoin":"1671509","getTokens":[]},"caTxList":[<bunch of txs we don't really care about here>]}}
```

Reply:
If we should drop invalid txs somewhere, it should be in `getTxHistory()` if I'm not mistaken

Reply:
so I think we don't have to change anything here, though the variables probably deserve renaming to make it clear we care about overall address infos and tx history is thrown away